src/jdk.httpserver/share/classes/sun/net/httpserver/UnmodifiableHeaders.java

```diff
@@ -82,7 +84,7 @@ public void clear() {
         }
 
         public Set<String> keySet() {
-            return Collections.unmodifiableSet (map.keySet());
+            return Collections.unmodifiableSet(map.keySet());
         }
```

Comment:
Should be able to just return "map.keySet()" here because map is already unmodifiable